src/osgPlugins/dae/ReaderWriterDAE.cpp

```diff
@@ -143,6 +143,9 @@ ReaderWriterDAE::readNode(const std::string& fname,
 
     std::string fileName( osgDB::findDataFile( fname, options ) );
     if( fileName.empty() ) return ReadResult::FILE_NOT_FOUND;
+    
+    pluginOptions.options = options ? static_cast<Options*>(options->clone(osg::CopyOp::SHALLOW_COPY)) : new Options;
```

Comment:
In recent OSG release you can use the osg::clone(options) template function instead of static_cast<Options*>(options->clone(osg::CopyOp::SHALLOW_COPY)).  The clone template function is defined in the include/osg/Object header.

Reply:
Thanks Robert. I've updated the code to use the osg::clone() function.